src/Model/GServer.php

```diff
@@ -1568,7 +1566,7 @@ private static function analyseRootBody($curlResult, array $serverdata, string $
 
 		// Using only body information we cannot safely detect a lot of systems.
 		// So we define a list of platforms that we can detect safely.
-		$valid_platforms = ['friendica', 'friendika', 'hubzilla', 'misskey', 'peertube', 'wordpress', 'write.as'];
+		$valid_platforms = ['friendica', 'friendika', 'diaspora', 'mastodon', 'hubzilla', 'misskey', 'peertube', 'wordpress', 'write.as'];
```

Comment:
This looks like a config or at least like a constant to me.

Reply:
This is something that a user mustn't configure, so a config file wouldn't be a good idea. And concerning using constants: We are using these strings at several places in the code. So when we change it here we had to change it at the other places in the system as well. So this should wait for the next release.